unit-tests/shared/src/test/scala/javalib/util/DefaultFormatterTest.scala

```diff
@@ -709,7 +709,7 @@ class DefaultFormatterTest {
   }
 
   @Test def formatForGeneralConversionType_bB(): Unit = {
-    val triple: Array[Array[Object]] = Array(
+    val triple: Array[Array[Any]] = Array[Array[Any]](
```

Comment:
Both sides and a cast are required?

Reply:
It definitely shouldn't be needed. I'll fix it

Reply:
Looks much better!